go.mod

```diff
@@ -170,7 +170,6 @@ require (
 )
 
 replace (
-	github.com/aws/aws-sdk-go-v2/config => github.com/aws/aws-sdk-go-v2/config v1.15.5 // same as buildkit
```

Comment:
This was on purpose to be consistent with BuildKit one: https://github.com/moby/buildkit/blob/4451e1be0e6889ffc56225e54f7e26bd6fdada54/go.mod#L12 so we don't upgrade inadvertently this dependency.

Reply:
In that case, let's add a more specific comment.. why do we suspect this particular dependency can break on upgrade? I was under impression that AWS SDK is not the kind of project that evolves rapidly, quite the opposite...

Reply:
> > In that case, let's add a more specific comment..
> 
> `// same as buildkit` looks clear enough to me but feel free to do so.

I read that as if buildkit repo is supposed have same replace directive, but it hasn't got one. So what you need to say is that the reason for replace statement is to pin the dependency and prevent upgrades.

> >  I was under impression that AWS SDK is not the kind of project that evolves rapidly, quite the opposite...
> 
> With [252 releases](https://github.com/aws/aws-sdk-go-v2) as of today I don't think so :sweat_smile:

98% of the SDK is generated code, most of the commits are additions for new API calls or new fields in existing API calls. Release count is probably very artificial also.